cms/static/js/models/settings/course_details.js

```diff
@@ -75,11 +82,45 @@ define(['backbone', 'underscore', 'gettext', 'js/models/validation_helpers', 'js
                         'The certificate available date must be later than the course end date.'
                     );
                 }
+
+
+                // TODO: Determine if certificates_display_behavior and certificate_available_date depend on each other,
```

Comment:
Is this still a TODO? Or leftover comments?

Reply:
That was taken care of. Removing :)